.gitignore

```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+dist
```

Comment:
I kept the `dist` folder because some of the code, especially the one in the copy and paste folder, needs to be accessible from the library repo. Don't just delete the folder, rather recommend a better approach instead of simply ignoring the folder.